solr/solr-ref-guide/modules/indexing-guide/pages/reindexing.adoc

```diff
@@ -165,6 +165,20 @@ Once the alias is in place and you are satisfied you no longer need the old data
 One advantage of this option is that you can switch back to the old collection if you discover problems our testing did not uncover.
 Of course this option can require more resources until the old collection can be deleted.
 
+=== Exporting/Importing Data from Solr
+
+Sometimes you don't want to run your full ETL pipeline to reindex into another collection, you just want to take the data in your existing collection, export it, and then import it back.
+
+There are a number of third party tools that do this, see https://solr.cool/ for more information.   However, if you want to use what ships with Solr then we have some options:
+
+1. Use `bin/solr export` with the JSON output format (`.json`), and the `bin/solr post` tool to post that data back.
```

Comment:
[Q] How would a user choose between these three options?  Or put differently - why would they choose one over the others?

If there's no strong differentiator between the three, is there value in mentioning them all individually?